### PR TITLE
add pull request manner

### PR DESCRIPTION
**Pull request by spacewander**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (hyperupcall) [inline .github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion
* To resolve merge conflicts, merge from the `main` branch instead of rebasing over `main`.
```

**Review by hyperupcall — APPROVED** [review]
lgtm

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (vanpipy) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: Once the merged from `main`, the commit tree will becomes messing. Is there a good practice here?

Comment (hyperupcall) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: @vanpipy There is something to be said about just merging instead of rebasing. But, looking at the [commit history](https://github.com/tj/git-extras/commits/main/), of this project, the Change Requests are _usually_ "Squashed and Merged" into `main` by the maintainers (unless they are not). So, in the common case, the commit tree will not become messed up

Comment (vanpipy) [reply .github/PULL_REQUEST_TEMPLATE.md:9]: I got the idea of that the "Squashed and Merged" is the fallback to squashed anyways to take us away from something disaster when using GitHub. Great 😆

**Comment by vanpipy** [comment]
LGTM

**Review by vanpipy — APPROVED** [review]
LGTM

**Pull request merged**
